tests/crypto/rand32/src/main.c

```diff
@@ -30,15 +30,17 @@
 
 void test_rand32(void)
 {
-	uint32_t gen, last_gen;
+	uint32_t gen, last_gen, tmp;
 	int rnd_cnt;
 	int equal_count = 0;
 	uint32_t buf[N_VALUES];
 
 	/* Test early boot random number generation function */
+	/* Add a condition about length less than uint32 */
```

Comment:
It is better to change the comments like below.

" Cover the case, where argument "length" is < size of "size_t". "

Reply:
Done